backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3263,4 +3263,60 @@ databaseChangeLog:
         - sql:
             remarks: Populate new `emails` column with value from patient `email`
             sql: |
-              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
\ No newline at end of file
+              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
+  - changeSet:
+      id: add-specimen_type-to-test_event
+      author: zedd@skylight.digital
+      comment: add specimen_type_id to test event
+      changes:
+        - tagDatabase:
+            tag: add-specimen_type-to-test_event
```

Comment:
It looks like the `id` here is somewhat inconsistent on 3273, 3287, and 3306. 3306 correctly references the full name of the column being added, whereas 3273 and 3287 mention an abbreviated version of the column being added (which leaves off the `_id` suffix). Not a deal-breaker by any stretch, but this may be something to make more consistent across these three.

Reply:
Technically the tag is unrelated to the `changeSet` id but agreed that we may want to make it match just for the sake of predictability (particularly since we're going to care about rollback tags when something has gone wrong).

Reply:
fixed in the other PR